### DISCUSSION TITLE
Disable rendering progress bar

### DISCUSSION THREAD

**Discussion by jaroslavknotek** [mitsuba-renderer/mitsuba3#527] [Q&A] [answered]
Is it possible to disable progress bar rendering? I tried setting log level to Warning  but that didn't help. 

After closer inspection I found out that the progress bar is rendered by a component ProgressReporter independently of log level (I found it [here](https://github.com/mitsuba-renderer/mitsuba3/blob/master/src/render/integrator.cpp#L144)). 

Is it possible to anyhow disable it? I render thousands of small images in which case progress bar doesn't really help. 

Thank you.

**Comment by njroussel** [comment]
Hi @jaroslavknotek 
No it currently can't be disabled.
I think hiding it at lower log levels makes sense. I'll put that on our to-do list.

**Comment by njroussel** [comment, marked as answer]
This has now been implemented https://github.com/mitsuba-renderer/mitsuba3/commit/c3edd68841fa145e5ba83c37345db413ca9c3a14.
